Logger/LogFactory.cs

```diff
@@ -2,10 +2,28 @@
 {
     public class LogFactory
     {
+        private string filePath;
```

Comment:
Field should be underscore PascalCase
```suggestion
        private string _FilePath;
```